src/testing/jest/jest-27-and-under/jest-runner.ts

```diff
@@ -105,8 +105,8 @@ export function includeTestFile(testPath: string, env: d.E2EProcessEnv) {
   return false;
 }
 
-export function getEmulateConfigs(testing: d.TestingConfig, flags: ConfigFlags) {
-  let emulateConfigs = testing.emulate.slice();
+export function getEmulateConfigs(testing: d.TestingConfig, flags: ConfigFlags): d.EmulateConfig[] {
+  let emulateConfigs = testing.emulate?.slice() ?? [];
```

Comment:
We technically backfill this property when we validate the testing config, but at this time, don't have a concept of a `ValidatedTestingConfig`